src/argo-bot.ts

```diff
@@ -34,6 +37,24 @@ const BotHelp = Object.freeze({
     Unlock: "removes lock held by current PR, allows other PR's to run bot",
 });
 
+export let BotHelpMessage = `
```

Comment:
Total nitpick, but I find the PR comment hard to read - compare https://github.com/ezcater/lab-kubernetes-cluster/pull/98#issuecomment-726122391 to the formatting on the post-plan comments in https://github.com/ezcater/terraform/pull/2057#issuecomment-726128343.

Is there any way to break this message up visually? Since it's going to be on every PR, maybe it should be limited to the diff command?

Reply:
I agree, this message isn't great. I just took the existing help message. I think that makes sense to only include the diff command.

Reply:
Actually, I think I'll implement that in another PR since this one is fairly large already. If I'm going to use some new text, I'd like to use a templating engine at the same time for clarity.

Reply:
I updated this to look more like the Atlantis comments.

https://github.com/ezcater/lab-kubernetes-cluster/pull/100

I am going to create a story that figures out the app name, but for now I just have a placeholder. I also expect that will just run a diff for that app.